### PR TITLE
Add ConTeXt support

### PR DESCRIPTION
**Pull request by flying-sheep**
I prefer ConTeXt to LaTeX, and support is pretty straightforward

### CODE REVIEW

**Comment by nschloe** [comment]
The linters fail; just run `make format` to get it right.

**Comment by flying-sheep** [comment]
Will do, thank you!

File: test/helpers.py [hunk 1]
```diff
@@ -64,11 +70,12 @@ def _compile(code):
...
+        context=["context", "--nonstopmode"],
+    )
     try:
-        subprocess.check_output(
-            ["pdflatex", "--interaction=nonstopmode", tex_file],
-            stderr=subprocess.STDOUT,
-        )
+        subprocess.check_output(cmds[flavor] + [tex_file], stderr=subprocess.STDOUT)
     except subprocess.CalledProcessError as e:
         print("pdflatex output:")
```
Comment (nschloe) [inline test/helpers.py]: You probably want to adapt this string too.

File: README.md [hunk 2]
```diff
@@ -120,23 +120,23 @@ to install/update.
...
-    ```latex
-    \usepackage[utf8]{inputenc}
-    \usepackage{fontspec}  % optional
-    \usepackage{pgfplots}
-    \pgfplotsset{compat=newest}
-    \usepgfplotslibrary{groupplots}
-    \usepgfplotslibrary{dateplot}
+   included in the header of your document. Get the code via:
+    ```python
+    import tikzplotlib
```
Comment (nschloe) [inline README.md]: Users will be too lazy to execute this command. Let's just keep the copy-pastable code in, and and add a code block for context, too, even if this means duplicating code from `Flavors`.

File: test/helpers.py (hunk 1 above)
Comment (flying-sheep) [reply test/helpers.py]: will do!

File: README.md (hunk 2 above)
Comment (flying-sheep) [reply README.md]: Yeah, problem is that it already diverged, but sure, it’s your project :wink:

**Comment by flying-sheep** [comment]
The coverage report said that I lowered the coverage, but when I click on the file, there’s an error and I can’t see where.

**Comment by nschloe** [comment]
Never mind the coverage.

**Comment by flying-sheep** [comment]
OK! Everything else passes now.

**Comment by nschloe** [comment]
LGTM, thanks!

**Comment by flying-sheep** [comment]
ooh, quick! thank you!

**Pull request merged**